samples/matlab_new/ignite.m

```diff
@@ -0,0 +1,104 @@
+function plotdata = ignite(g)
```

Comment:
This works, but I have a more general comment about the folder structure.

Do you think that it makes sense to create a similar folder structure to the one used for Python examples?

Reply:
The # of examples in the Matlab toolbox is much fewer than Python examples so most of the folders won't be populated. I think I will do this in a future feature update where I converted extra Python examples to Matlab ones.